src/main/java/com/epam/spring/homework3/controller/PlaceController.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.spring.homework3.controller;
+
+import com.epam.spring.homework3.controller.dto.PlaceDTO;
+import com.epam.spring.homework3.service.PlaceService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@Slf4j
+@RequiredArgsConstructor
+public class PlaceController {
+    private final PlaceService placeService;
+
+    @GetMapping("/place/{placeId}")
```

Comment:
please use @RequestMapping("/place") instead of duplicating the same url in each method. And check other controllers to this issue